Jenkinsfile

```diff
@@ -7,6 +7,10 @@ pipeline {
 
     tools {nodejs 'node6.11.3'}
 
+    enviroment {
+        DISABLE_SSL = 'true'
```

Comment:
Die Variable bitte noch in .env eintragen. Und dann .env in .env.sample umbenennen, damit die nicht aus Versehen produktiv geht.